src/tuned/seapath-rt-host-variables.conf.j2

```diff
@@ -0,0 +1 @@
+isolated_cores={{ isolcpus | default('') }}
```

Comment:
Why make that in a separate template file ?
It is simpler to include this line in `src/tuned/tuned.conf` and templatize the entire file at once.

Reply:
I guess it's just the usual way of tuned doing things. Usually the profiles are static (in /usr...) and customizables with variables in /etc
That's how @yairpod first submitted the PR, by only modifying a variable file.
Since we created our own Seapath profile, we did the same things, a "static" profile in tuned.conf and variables in a separate file.

Reply:
I merged both files